src/main/java/in/org/projecteka/hdaf/link/link/model/PatientLinkResponse.java

```diff
@@ -0,0 +1,21 @@
+package in.org.projecteka.hdaf.link.link.model;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
nic: I think there is a way to configure this at global level, instead of doing this for each and every class. We can do the separately also.